flexmeasures/ui/views/new_dashboard.py

```diff
@@ -0,0 +1,55 @@
+from flask import request, current_app
+from flask_security import login_required
+from flask_security.core import current_user
+from bokeh.resources import CDN
+
+from flexmeasures.data.config import db
+from flexmeasures.ui.views import flexmeasures_ui
+from flexmeasures.ui.utils.view_utils import render_flexmeasures_template, clear_session
+from flexmeasures.data.models.generic_assets import get_center_location_of_assets
+from flexmeasures.data.services.asset_grouping import (
+    AssetGroup,
+    get_asset_group_queries,
+)
+
+
+# Dashboard (default root view, see utils/app_utils.py)
+@flexmeasures_ui.route("/dashboard")
+@login_required
+def new_dashboard_view():
+    """Dashboard view.
+    This is the default landing page.
+    It shows a map with the location of all of the user's assets,
+    as well as a breakdown of the asset types in the user's portfolio.
+    TODO: Assets for which the platform has identified upcoming balancing opportunities are highlighted.
+    """
+    msg = ""
+    if "clear-session" in request.values:
+        clear_session()
+        msg = "Your session was cleared."
+
+    aggregate_groups = ["renewables", "EVSE"]
+    asset_groups = get_asset_group_queries(custom_additional_groups=aggregate_groups)
+    map_asset_groups = {}
+    for asset_group_name in asset_groups:
+        asset_group = AssetGroup(asset_group_name)
+        map_asset_groups[asset_group_name] = asset_group
```

Comment:
Some groups contain only generic assets without a physical location (e.g. former Markets), which still show up as an empty layer on the map you can toggle (without effect). I suggest removing those groups from the map at this point.

Reply:
Yes. I'd remove them from the whole dashboard. Technically, I should display only Assets which have at least one sensor with `is_consumer=True` or `is_producer=True`.

Reply:
What do you want to achieve by doing that: display only locations of generic assets with at least 1 power sensor? If so, I recommend checking the sensor `unit` instead.

Reply:
Yes. Perhaps add a fixed set to `utils/unit_utils.py` for now. We should really make a ticket to start using [pint](https://pint.readthedocs.io/en/stable/).

Reply:
I feel my approach is the more generic one at the moment.

That pint package sounds very useful to us. Can we save combined unit names (e.g. "kWh" or "m/s") in the `Sensor.unit` column which it can understand?

Reply:
My approach isn't going to work well, as attributes are only on sensors who lived as assets before. We don't yet support them on newly-formed sensors. I have to go with your `unit` idea then.